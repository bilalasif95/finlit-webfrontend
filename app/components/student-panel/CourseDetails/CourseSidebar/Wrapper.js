import styled from 'styled-components';

const Wrapper = styled.div`
  border: 1px solid #d8d8d8;
  border-radius: 0.625rem;
  margin: 30px 0 16px;
  width: 100%;
  .top_detail {
    border-bottom: 1px solid #d8d8d8;
    display: flex;
    flex-direction: column;
    padding: 1.5rem;
    .prices {
      align-items: flex-start;
      display: flex;
      flex-direction: column;
      justify-content: flex-start;
      margin: 0 0 0.5rem;
      width: 100%;
      .old_price {
        align-items: center;
        display: flex;
        justify-content: flex-start;
      }
      h4 {
        color: #404041;
        font-size: 1.5rem;
        font-weight: 600;
        margin: 0 0 0.5rem;
      }
      del {
        color: #8b8b8b;
        font-size: 0.875rem;
        margin: 0 1rem 0 0;
      }
      span {
        background-color: #ffd416;
        border-radius: 0.188rem;
        color: #404041;
        font-size: 0.625rem;
        font-weight: 550;
        padding: 0.25rem 0.5rem;
      }
    }
    .rating {
      align-items: center;
      display: flex;
      justify-content: flex-start;
      margin: 1rem 0;
      width: 100%;
      span {
        font-size: 0.875rem;
      }
      p {
        color: #484848;
        margin: 0 0 0 0.5rem;
      }
      .stars {
        margin-left: 0.5rem;
        svg {
          g {
            path:last-child {
              fill: #fbbf2e;
            }
          }
        }
      }
    }
    .enroll_now {
      background-color: transparent;
      border: 1px solid #707070;
      color: #8b8b8b;
      svg {
        path {
          stroke: #8b8b8b;
        }
      }
    }
    .add_cart {
      background-color: #46a135;
      border: 1px solid #46a135;
      color: #fff;
      svg {
        path {
          stroke: #fff;
        }
      }
    }
    button {
      border-radius: 0.313rem;
      color: #fff;
      font-size: 0.875rem;
      font-weight: 550;
      line-height: 17px;
      font-size: 0.875rem;
      padding: 0.85rem;
      text-align: center;
      text-decoration: none;
      margin: 1rem 0 0;
      padding: 0.85rem;
      -webkit-touch-callout: none;
      -webkit-user-select: none;
      -khtml-user-select: none;
      -moz-user-select: none;
      -ms-user-select: none;
      user-select: none;
      -webkit-tap-highlight-color: transparent;
      position: relative;
      width: 100%;
      &:hover,
      &:focus {
        box-shadow: none;
      }
    }
    button, span {
      transition: 200ms;
    }
    
    .text {
      width: 100%;
    }
    
    .icon {
      position: absolute;
      transform: translateX(0);
      height: 1.25rem;
      width: 1.25rem;
      display: flex;
      align-items: center;
      justify-content: center;
    }
    
     svg {
      font-size: 1.25rem;
    }
    button:hover .text {
      color: transparent;
    }
    
    button:hover .icon {
      width: 90%;
      transform: translateX(0);
    }
  }
  .details {
    padding: 1.5rem;
    h4 {
      color: #404041;
      font-size: 1.25rem;
      line-height: 24px;
      margin: 0 0 1rem;
    }
    ul {
      margin: 1rem 0 0;
      padding: 0;
      li {
        align-items: center;
        display: flex;
        color: #484848;
        font-size: 0.875rem;
        line-height: 17px;
        list-style: none;
        justify-content: flex-start;
        padding: 0 0 1rem;
        svg {
          font-size: 1.25rem;
          margin: 0 1rem 0 0;
        p {
          margin: 0;
          width: 100%;
        }
      }
    }
    .form-group {
      margin: 1.5rem 0 1rem;
      position: relative;
      label {
        color: #484848;
        font-size: 0.875rem;
      }
      .form-control {
        border-top-right-radius: 0.25rem !important;
        border-bottom-right-radius: 0.25rem !important;
        padding: 0.6rem 1rem;
        &:focus {
          box-shadow: none;
        }
        &::placeholder {
          color: #e6e6e6;
        }
      }
      button {
        background-color: #46a135;
        border: 1px solid #46a135;
        border-top-right-radius: 0.313rem;
        border-bottom-right-radius: 0.313rem;
        border-top-left-radius: 0;
        border-bottom-left-radius: 0;
        color: #fff;
        font-size: 0.875rem;
        font-weight: 550;
        line-height: 17px;
        padding: 0.72rem;
        position: absolute;
        right: 0;
        top: 0;
        width: 100px;
        &:hover {
          background-color: #46a135;
          border: 1px solid #46a135;
        }
        &:focus {
          background-color: #46a135;
          border: 1px solid #46a135;
          box-shadow: none;
        }
      }
    }
  }

  @media (max-width: 991px) {
    .details {
      padding: 1rem;
      .form-group {
        margin: 0.5rem 0 0;
      }
    }
  }
  @media (max-width: 960px) and (min-width: 768px) {
    margin: 0 0 1.5rem;
  }
  @media (max-width: 767px) {
    margin: 1rem 0;
  }
`;

export default Wrapper;
