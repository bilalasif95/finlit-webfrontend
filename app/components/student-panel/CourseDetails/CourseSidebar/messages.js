/*
 * Cart Checkout Sidebar Messages
 *
 * This contains all the text for the CartCheckoutSidebar component.
 */
import { defineMessages } from 'react-intl';

export const scope = 'boilerplate.components.CartCheckoutSidebar';

export default defineMessages({
  EnrollNow: {
    id: `${scope}.EnrollNow`,
    defaultMessage: `Enroll Now`,
  },
  AddtoCart: {
    id: `${scope}.AddtoCart`,
    defaultMessage: `Add to Cart`,
  },
  Details: {
    id: `${scope}.Details`,
    defaultMessage: `This course includes`,
  },
  Apply: {
    id: `${scope}.Apply`,
    defaultMessage: `Apply`,
  },
});
