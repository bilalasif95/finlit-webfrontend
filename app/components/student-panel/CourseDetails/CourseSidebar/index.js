/*
 * Cart Checkout Sidebar Component
 */
import React, { useState } from 'react';
import { FormattedMessage } from 'react-intl';
import { FormGroup, Input, Progress, Button, Label } from 'reactstrap';
import axios from 'axios';
import PropTypes from 'prop-types';
import { ToastContainer, toast } from 'react-toastify';
import 'react-toastify/dist/ReactToastify.css';
import history from 'utils/history';
import { Link } from 'react-router-dom';
import { GoDeviceCameraVideo } from 'react-icons/go';
import { BiTimeFive } from 'react-icons/bi';
import { FaAward } from 'react-icons/fa';
import { BsTagFill } from 'react-icons/bs';
import { SiAirplayvideo } from 'react-icons/si';
import { RiStarSFill } from 'react-icons/ri';
import { GrCart, GrCirclePlay, GrCertificate } from 'react-icons/gr';
import { HiUser, HiOutlineArrowNarrowRight } from 'react-icons/hi';
import Wrapper from './Wrapper';
import messages from './messages';
import { API } from '../../../../config/config';
import { axiosHeader } from '../../../../utils/axiosHeader';

function CourseSidebar(props) {
  const [loader, setLoader] = useState(false);
  // const { detail } = props;
  // const dataDetails = detail.data;

  const handleAddToCart = () => {
    setLoader(true);
    const data = {
      productType: detail.type,
      productId: detail.id,
      price: dataDetails && dataDetails.price,
    };

    axios
      .post(`${API}api/cart/addTocart`, data, axiosHeader)
      .then(() => {
        history.push('/cart');
        setLoader(false);
      })
      .catch(err => {
        setLoader(false);
        toast.error(
          err.response
            ? err.response && err.response.message
            : 'Message Not Readable',
        );
      });
  };

  return (
    <Wrapper>
      <div className="sidebar">
        <div className="intro">
          <div className="top_detail">
            <div className="prices">
              {/* <h4>${dataDetails && dataDetails.price}</h4>
          <del>${dataDetails && dataDetails.price}</del> */}
              <h4>$44.99</h4>
              <div className="old_price">
                <del>$100</del>
                <span>50% off</span>
              </div>
            </div>
            {/* <div className="rating">
          <p>4.7</p>
          <div className="stars">
            <RiStarSFill />
            <RiStarSFill />
            <RiStarSFill />
            <RiStarSFill />
            <RiStarSFill />
          </div>
          <p>(76,876 ratings)</p>
        </div> */}
            {/* <Button onClick={() => handleAddToCart()} disabled={loader}>
          {loader ? 'Loading..' : <FormattedMessage {...messages.AddtoCart} />}
        </Button> */}
            <Button className="add_cart">
              <span className="icon"><GrCart /></span>
              <span className="text">
                <FormattedMessage {...messages.AddtoCart} />
              </span>
            </Button>
            {/* <div className="test">
          <button className="noselect"><span className="text">Delete</span></button>
        </div> */}
            <Button className="enroll_now">
              <span className="icon"><HiOutlineArrowNarrowRight /></span>
              <span className="text">
                <FormattedMessage {...messages.EnrollNow} />
              </span>
            </Button>
          </div>
          <div className="details">
            <h4>
              <FormattedMessage {...messages.Details} />
            </h4>
            <ul className="list">
              <li>
                <GrCirclePlay />
                <p>2 hours</p>
              </li>
              <li>
                <GoDeviceCameraVideo />
                <p>8 Lectures</p>
              </li>
              <li>
                <HiUser />
                <p>1200 Enroll</p>
              </li>
              <li>
                <BiTimeFive />
                <p>Full lifetime access</p>
              </li>
              <li>
                {/* <GrCertificate /> */}
                <FaAward />
                <p>Certificate of completion</p>
              </li>
            </ul>
            {/* <FormGroup>
          <Input
            type="text"
            name="coupon"
            id="coupon"
            placeholder="Enter Coupon"
          />
          <Button>
            <FormattedMessage {...messages.Apply} />
          </Button>
        </FormGroup> */}
          </div>
        </div>
        <div className="course_status">
          <div className="progress_chart">
            <div className="progress_time">
              <h4>Course Progress</h4>
              <p>Expend time 8.3hr</p>
            </div>
            <div className="chart">
              <img src="https://cdn.discordapp.com/attachments/548125906472796170/905387362706681866/unknown.png" alt="chart" height="100%" width="100%" />
            </div>
          </div>
          <div className="progress_session">
            <div className="session">
              <p>Session 1 progress</p>
              <p>50/100</p>
            </div>
            <div className="progress_bar">
            <Progress value={50} />
            </div>
          </div>
          <div className="quiz">
            <div className="inner_box">
              <div className="title">
                <h4>Quiz 1</h4>
                </div>
                <div classname="quiz_box">
                  <h6>The acronym PC stands for?</h6>
                  <ul>
                    <li>
                      <Label>
                        <Input type="radio" />
                        
                      </Label>
                    </li>
                  </ul>
                  </div>
              </div>
            </div>
        </div>
      </div>
      <ToastContainer />
    </Wrapper>
  );
}
// CourseSidebar.propTypes = {
//   detail: PropTypes.any,
// };
export default CourseSidebar;
