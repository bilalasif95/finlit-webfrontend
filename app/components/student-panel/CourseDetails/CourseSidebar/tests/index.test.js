import React from 'react';
import renderer from 'react-test-renderer';
import { IntlProvider } from 'react-intl';
import { Provider } from 'react-redux';
// import { browserHistory, BrowserRouter } from 'react-router-dom';
import { Router } from 'react-router-dom';
import history from 'utils/history';
import CourseSidebar from '../index';
// import configureStore from '../../../../../configureStore';
import Store2 from '../../../../../containers/reduxSetup/store';
describe('<CourseSidebar />', () => {
  let store;

  beforeAll(() => {
    // store = configureStore({}, browserHistory);
    store = Store2;
  });

  it('should render and match the snapshot', () => {
    const renderedComponent = renderer
      .create(
        <Provider store={store}>
          <IntlProvider locale="en">
            {/* <BrowserRouter> */}
            <Router history={history}>
              {/* <CourseSidebar detail={{ data: {} }} /> */}
              <CourseSidebar />
              {/* </BrowserRouter> */}
            </Router>
          </IntlProvider>
        </Provider>,
      )
      .toJSON();

    expect(renderedComponent).toMatchSnapshot();
  });
});
