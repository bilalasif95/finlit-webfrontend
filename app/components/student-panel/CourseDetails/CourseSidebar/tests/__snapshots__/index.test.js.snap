// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HackathonSidebar /> should render and match the snapshot 1`] = `
<div
  className="Wrapper-sc-1py71q8-0 coGmPC"
>
  <div
    className="sidebar"
  >
    <div
      className="intro"
    >
      <div
        className="top_detail"
      >
        <div
          className="prices"
        >
          <h4>
            $44.99
          </h4>
          <div
            className="old_price"
          >
            <del>
              $100
            </del>
            <span>
              50% off
            </span>
          </div>
        </div>
        <button
          aria-label={null}
          className="add_cart btn btn-secondary"
          onClick={[Function]}
        >
          <span
            className="icon"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M5,5 L22,5 L20,14 L7,14 L4,2 L0,2 M7,14 L8,18 L21,18 M19,23 C18.4475,23 18,22.5525 18,22 C18,21.4475 18.4475,21 19,21 C19.5525,21 20,21.4475 20,22 C20,22.5525 19.5525,23 19,23 Z M9,23 C8.4475,23 8,22.5525 8,22 C8,21.4475 8.4475,21 9,21 C9.5525,21 10,21.4475 10,22 C10,22.5525 9.5525,23 9,23 Z"
                fill="none"
                stroke="#000"
                strokeWidth="2"
              />
            </svg>
          </span>
          <span
            className="text"
          >
            <span>
              Add to Cart
            </span>
          </span>
        </button>
        <button
          aria-label={null}
          className="enroll_now btn btn-secondary"
          onClick={[Function]}
        >
          <span
            className="icon"
          >
            <svg
              fill="none"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M17 8l4 4m0 0l-4 4m4-4H3"
                strokeLinecap="round"
                strokeLinejoin="round"
                strokeWidth="2"
              />
            </svg>
          </span>
          <span
            className="text"
          >
            <span>
              Enroll Now
            </span>
          </span>
        </button>
      </div>
      <div
        className="details"
      >
        <h4>
          <span>
            This course includes
          </span>
        </h4>
        <ul
          className="list"
        >
          <li>
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M12,22 C17.5228475,22 22,17.5228475 22,12 C22,6.4771525 17.5228475,2 12,2 C6.4771525,2 2,6.4771525 2,12 C2,17.5228475 6.4771525,22 12,22 Z M9.5,15.5 L15.5,12 L9.5,8.5 L9.5,15.5 Z M10.5,13.5 L12.5,12 L10.5,10.5 L10.5,13.5 Z"
                fill="none"
                stroke="#000"
                strokeWidth="2"
              />
            </svg>
            <p>
              2 hours
            </p>
          </li>
          <li>
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 16 16"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M15.2 2.09L10 5.72V3c0-.55-.45-1-1-1H1c-.55 0-1 .45-1 1v9c0 .55.45 1 1 1h8c.55 0 1-.45 1-1V9.28l5.2 3.63c.33.23.8 0 .8-.41v-10c0-.41-.47-.64-.8-.41z"
                fillRule="evenodd"
              />
            </svg>
            <p>
              8 Lectures
            </p>
          </li>
          <li>
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 20 20"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clipRule="evenodd"
                d="M10 9a3 3 0 100-6 3 3 0 000 6zm-7 9a7 7 0 1114 0H3z"
                fillRule="evenodd"
              />
            </svg>
            <p>
              1200 Enroll
            </p>
          </li>
          <li>
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M12,2C6.486,2,2,6.486,2,12s4.486,10,10,10s10-4.486,10-10S17.514,2,12,2z M12,20c-4.411,0-8-3.589-8-8s3.589-8,8-8 s8,3.589,8,8S16.411,20,12,20z"
              />
              <path
                d="M13 7L11 7 11 12.414 14.293 15.707 15.707 14.293 13 11.586z"
              />
            </svg>
            <p>
              Full lifetime access
            </p>
          </li>
          <li>
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 384 512"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M97.12 362.63c-8.69-8.69-4.16-6.24-25.12-11.85-9.51-2.55-17.87-7.45-25.43-13.32L1.2 448.7c-4.39 10.77 3.81 22.47 15.43 22.03l52.69-2.01L105.56 507c8 8.44 22.04 5.81 26.43-4.96l52.05-127.62c-10.84 6.04-22.87 9.58-35.31 9.58-19.5 0-37.82-7.59-51.61-21.37zM382.8 448.7l-45.37-111.24c-7.56 5.88-15.92 10.77-25.43 13.32-21.07 5.64-16.45 3.18-25.12 11.85-13.79 13.78-32.12 21.37-51.62 21.37-12.44 0-24.47-3.55-35.31-9.58L252 502.04c4.39 10.77 18.44 13.4 26.43 4.96l36.25-38.28 52.69 2.01c11.62.44 19.82-11.27 15.43-22.03zM263 340c15.28-15.55 17.03-14.21 38.79-20.14 13.89-3.79 24.75-14.84 28.47-28.98 7.48-28.4 5.54-24.97 25.95-45.75 10.17-10.35 14.14-25.44 10.42-39.58-7.47-28.38-7.48-24.42 0-52.83 3.72-14.14-.25-29.23-10.42-39.58-20.41-20.78-18.47-17.36-25.95-45.75-3.72-14.14-14.58-25.19-28.47-28.98-27.88-7.61-24.52-5.62-44.95-26.41-10.17-10.35-25-14.4-38.89-10.61-27.87 7.6-23.98 7.61-51.9 0-13.89-3.79-28.72.25-38.89 10.61-20.41 20.78-17.05 18.8-44.94 26.41-13.89 3.79-24.75 14.84-28.47 28.98-7.47 28.39-5.54 24.97-25.95 45.75-10.17 10.35-14.15 25.44-10.42 39.58 7.47 28.36 7.48 24.4 0 52.82-3.72 14.14.25 29.23 10.42 39.59 20.41 20.78 18.47 17.35 25.95 45.75 3.72 14.14 14.58 25.19 28.47 28.98C104.6 325.96 106.27 325 121 340c13.23 13.47 33.84 15.88 49.74 5.82a39.676 39.676 0 0 1 42.53 0c15.89 10.06 36.5 7.65 49.73-5.82zM97.66 175.96c0-53.03 42.24-96.02 94.34-96.02s94.34 42.99 94.34 96.02-42.24 96.02-94.34 96.02-94.34-42.99-94.34-96.02z"
              />
            </svg>
            <p>
              Certificate of completion
            </p>
          </li>
        </ul>
      </div>
    </div>
    <div
      className="course_status"
    >
      <div
        className="progress_chart"
      >
        <div
          className="progress_time"
        >
          <h4>
            Course Progress
          </h4>
          <p>
            Expend time 8.3hr
          </p>
        </div>
        <div
          className="chart"
        >
          <img
            alt="chart"
            height="100%"
            src="https://cdn.discordapp.com/attachments/548125906472796170/905387362706681866/unknown.png"
            width="100%"
          />
        </div>
      </div>
      <div
        className="progress_session"
      >
        <div
          className="session"
        >
          <p>
            Session 1 progress
          </p>
          <p>
            50/100
          </p>
        </div>
        <div
          className="progress_bar"
        >
          <div
            className="progress"
            style={Object {}}
          >
            <div
              aria-valuemax={100}
              aria-valuemin={0}
              aria-valuenow={50}
              className="progress-bar"
              role="progressbar"
              style={
                Object {
                  "width": "50%",
                }
              }
            />
          </div>
        </div>
      </div>
      <div
        className="quiz"
      >
        <div
          className="inner_box"
        >
          <div
            className="title"
          >
            <h6>
              Quiz 1
            </h6>
          </div>
          <div
            className="quiz_box"
          >
            <h6>
              The acronym PC stands for?
            </h6>
            <ul>
              <li>
                <div
                  className="form-check"
                >
                  <label
                    className="form-check-label"
                  >
                    <input
                      className="form-check-input"
                      name="radio1"
                      onChange={[Function]}
                      type="radio"
                      value="radio1"
                    />
                     
                    <div
                      className="check_mark"
                    >
                      <span>
                        1
                      </span>
                      <svg
                        fill="none"
                        height="1em"
                        stroke="currentColor"
                        strokeLinecap="round"
                        strokeLinejoin="round"
                        strokeWidth="2"
                        style={
                          Object {
                            "color": undefined,
                          }
                        }
                        viewBox="0 0 24 24"
                        width="1em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <polyline
                          points="20 6 9 17 4 12"
                        />
                      </svg>
                    </div>
                    <p>
                      Private Computer
                    </p>
                  </label>
                </div>
              </li>
              <li>
                <div
                  className="form-check"
                >
                  <label
                    className="form-check-label"
                  >
                    <input
                      className="form-check-input"
                      name="radio2"
                      onChange={[Function]}
                      type="radio"
                      value="radio2"
                    />
                     
                    <div
                      className="check_mark"
                    >
                      <span>
                        2
                      </span>
                      <svg
                        fill="none"
                        height="1em"
                        stroke="currentColor"
                        strokeLinecap="round"
                        strokeLinejoin="round"
                        strokeWidth="2"
                        style={
                          Object {
                            "color": undefined,
                          }
                        }
                        viewBox="0 0 24 24"
                        width="1em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <polyline
                          points="20 6 9 17 4 12"
                        />
                      </svg>
                    </div>
                    <p>
                      Personal Computer
                    </p>
                  </label>
                </div>
              </li>
              <li>
                <div
                  className="form-check"
                >
                  <label
                    className="form-check-label"
                  >
                    <input
                      className="form-check-input"
                      name="radio3"
                      onChange={[Function]}
                      type="radio"
                      value="radio3"
                    />
                     
                    <div
                      className="check_mark"
                    >
                      <span>
                        3
                      </span>
                      <svg
                        fill="none"
                        height="1em"
                        stroke="currentColor"
                        strokeLinecap="round"
                        strokeLinejoin="round"
                        strokeWidth="2"
                        style={
                          Object {
                            "color": undefined,
                          }
                        }
                        viewBox="0 0 24 24"
                        width="1em"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <polyline
                          points="20 6 9 17 4 12"
                        />
                      </svg>
                    </div>
                    <p>
                      Personal Compact
                    </p>
                  </label>
                </div>
              </li>
            </ul>
          </div>
        </div>
        <div
          className="quiz_step"
        >
          <p>
            Question 1 of 3
          </p>
          <button
            aria-label={null}
            className="btn_next btn btn-secondary"
            onClick={[Function]}
            type="button"
          >
            Next
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="Toastify"
  />
</div>
`;
